x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/enrich/action/EnrichStatsAction.java

```diff
@@ -103,12 +123,13 @@ public boolean equals(Object o) {
             if (o == null || getClass() != o.getClass()) return false;
             Response response = (Response) o;
             return executingPolicies.equals(response.executingPolicies) &&
-                coordinatorStats.equals(response.coordinatorStats);
+                coordinatorStats.equals(response.coordinatorStats) &&
+                cacheStats.equals(response.cacheStats);
```

Comment:
cacheStats could be end up null in mixed clusters

Reply:
Good spot, I will fix this.

Reply:
Fixed: 55e6e2424b7c680324dacf04a44e1b586220b72c